NEW_README.md

```diff
@@ -0,0 +1,67 @@
+    * Replace pe_dir with a valid pe_dir (past versions can be found at http://enterprise.delivery.puppetlabs.net/archives/releases/)
+        * Local dir or http url of a pe tarball
+    * Replace pe_ver with the pe version to install
+        * The version string after ‘enterprise’ and before the platform name/version
+* If you are using an aws puppet-bastion account
+    * Assume the ESO role
+        * aws sts assume-role --role-arn arn:aws:iam::028918822489:role/\<user> --role-session-name example --serial-number arn:aws:iam::103716600232:mfa/\<user> --token-code \<mfa token code>
+        * Place the returned creds in your .fog file
+            * Beaker-aws will look for the aws_access_key_id, aws_secret_access_key and aws_session_token keys
+* Execute:
+    * export BEAKER_HOSTS=\<your hosts file>
+    * bundle install
+    * bundle exec rake performance_gatling (takes up to 15 minutes)
+
+### Record a Gatling run:
+* Ensure you are still SSH'd to the Gatling driver node (metrics) using the -Y argument `ssh -Y root@<driver-host>`
+* Execute `cd ~/gatling-puppet-load-test/proxy-recorder`
+* Execute `sh ./launch-gatling-proxy.sh`
+* Follow the steps from the script
+* In the Gatling recorder GUI:
+    * Change the ‘listening port’ to 7000
+    * Change ‘class name’ to a unique value with no spaces such as ‘HoytApples’ - remember this, as this is the \<GatlingClassName> in subsequent steps.
+    * Press start
+* Back in the cmd line on the Gatling driver node, press “enter”
+* Copy the command to execute on the agent node
+* SSH into the agent node and:  (Ignore instructions from the script)
+    * Paste in the command
+    * Replace value for --server with the FQDN of the MOM
+    * Replace value for --http_proxy_host with the IP address of the Gatling driver node
+    * Change the value for --http_proxy_port to 7000
+    * Execute the command
+* Ensure that traffic was captured (rows should appear in the ‘executed events’ section of the Gatling recorder GUI)
+* Press ‘Stop and Save’
+* Close the GUI
+* On the Gatling driver node, press enter
+* Continue to follow the script.
+    * When asked for the certname prefix enter any value containing the string ‘agent’. For example perf-agent-0
+    * When asked for Puppet classes, enter the configuration which defaults to: role::by_size::large
+        * You can use a different puppet class by specifying it as the `PUPPET_SCALE_CLASS` environment variable for the performance_gatling rake task.
+### Getting a pre-recorded run started
+For the following steps, GatlingClassName is the value entered into the ClassName field during the recording step.
+From root of the gatling-puppet-load-test source dir on the Gatling driver (metrics):
+* First time only:
+    * edit /usr/share/sbt/conf/sbtopts and change “-mem” to 2048 (and uncomment)
+* For each new simulation:
+    * `cd ~/gatling-puppet-load-test/simulation-runner`
+    * `export GEM_SOURCE=http://rubygems.delivery.puppetlabs.net`
```

Comment:
rubygems.delivery.puppetlabs.net won't work at some point in the near future (should be artifactory).

Reply:
True, this sprint we're actually working on automating the gatling record and playback steps so I will make sure that we update it to use artifactory.  Thanks!